src/GraphQL.Tests/GraphQL.Tests.csproj

```diff
@@ -19,6 +19,16 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
revert changes?

Reply:
I'd leave it - ok?  It's useful and only affects the test project.  Ideally this would be added into the Test.props file so if a test hangs in any test project, we can identify it without 3 hours of work (like I did this morning).

Reply:
OK, move into tests.pros; ` <None Remove="xunit.runner.json" />` may be removed.

Reply:
Done